easybuild/easyconfigs/g/GULP/GULP-6.0-foss-2020a.eb

```diff
@@ -0,0 +1,51 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on GULP-5.1-intel-2019a.eb
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'GULP'
+version = '6.0'
+
+homepage = 'https://gulp.curtin.edu.au/gulp/'
+description = """GULP is a program for performing a variety of types of simulation on materials
+ using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs
+ and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {
+    'openmp': True,
+    'usempi': True,
+}
+
+# download requires registration, https://gulp.curtin.edu.au/gulp/request.cfm?rel=download
+sources = [SOURCELOWER_TGZ]
+patches = ['gulp-6.0-fftw-blas-linking-gcc-9.3.patch']
+checksums = [
+    'b63cff96300356728ee641e7bac8f065a01f108a2289711c3d3d11d4050527f6',  # gulp-6.0.tgz
+    'c778df9d6b8d31594be81a45271387ed885753e54e4c5009ee0f55382aa6a4d6',  # gulp-6.0-fftw-blas-linking-gcc-9.3.patch
+]
+
+start_dir = 'Src'
+
+build_cmd = "./mkgulp -f -m -c gfortran"
```

Comment:
As GULP isn't semi-closed source, it might make sense to include a short comment about the used options. IIRC `-c` is the compiler to use and `-m` is for parallel.
Not really required but I think it makes it easier for anyone checking on this later.

Reply:
I have added a comment regarding the meaning of the various flags plus how to use the serial and parallel run for GULP.
This way, it should be possible for at least the person who installs it to see a pure serial build is not required.